jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -0,0 +1,368 @@
+
+. From the CircleCI web app, click **Dashboard** in the sidebar
+. Use the project filter dropp-down to choose your project
+. Click btn:[Project Settings] and then choose **Environment Variables**
+. Click btn:[Add Environment Variable] button to enter a name and value of the new environment variable.
++
+image::deploy/add-env-var.png[Screenshot showing add environment variable popup]
+. Repeat step four for the second environment variable
+
+[#set-up-config]
+== 4. Set up your CircleCI configuration
+
+Take a look at the `.circleci/config.yml` for the link:https://github.com/CircleCI-Public/sagemaker-deploy-examples/blob/main/.circleci/config.yml[sample project]. There is a block of pipeline parameters at the top. You need to update the defaults with your values.
+
+[source,yaml]
+----
+parameters:
+  bucket:
+    type: string
+    default: circleci-sagemaker-pipeline
+  model_desc:
+    type: string
+    default: "Kitten Classifier allowing us to distinguish between giraffes and kittens."
+  model_name:
+    type: string
+    default: kitten-classifier
+  project_id:
+    type: string
+    default: "e47ee9b0-446f-44cf-bec8-5407ceb06930"
+  region_name:
+    type: string
+    default: us-east-1
+----
+
+Make the following updates:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+|Parameter
+|Default
+
+|`bucket`
+|The bucket you set up in the Role Policy
+
+|`model_desc`
+|A description of your model
+
+|`model_name`
+|The name of the model
+
+|`project_id`
+|The CircleCI project ID, you can find this on the Project Settings page in the CircleCI web app
+
+|`region_name`
+|The region, for example, `us-east-1`
+|===
+
+[#models]
+== 4. Build a model package version
+
+The sample app used in this how-to guide uses a model commonly found in AWS documentation, **Abalone**. It has just been renamed. If you already have your own model, feel free to adapt the configuration file to use that one instead. All you will need to do is update the `model_name` parameter.
+
+Assuming you are using our example repository, follow these steps to build a new model package version:
+
+. There is a workflow (`model-train`) configured to run on the branch `model-train`. Checkout the `model-train` branch, and push it up to GitHub
+. The `model-train` workflow will make a new model package version in the model registry. If the model package doesn't already exist, it will create it.
+
+Every time you run this workflow by pushing to the `model-train` branch, anew model version is created.
+
+[#using-tables]
+== 5. Use the orb
+
+Let's break down the `deploy-model-through-to-prod workflow`. Our first job is `aws-sagemaker/create_model`. This job creates a model from your latest Model Package in the registry. This will be what we then deploy to the inference endpoints. In general, `create-model` just needs to be called one time at the beginning of your workflow:
```

Comment:
I don't know what is meant by "inference endpoints" here but if it will be clear to an experienced reader that's fine

Reply:
Yes I think it's safe to assume that this reference will be clear to the audience!